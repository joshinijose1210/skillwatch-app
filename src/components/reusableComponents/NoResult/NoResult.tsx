import { Text } from '@medly-components/core';
import { NoResultProps } from './types';

export const NoResult = ({ title }: NoResultProps) => {
    return (
        <Text textVariant="h5" textAlign="center">
            {title}
        </Text>
    );
};
