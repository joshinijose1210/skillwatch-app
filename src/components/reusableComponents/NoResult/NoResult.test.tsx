import { render } from '@testing-library/react';
import { NoResult } from './NoResult';

describe('KpiDescriptionEditor component', () => {
    it('should take snapshot', () => {
        const { container } = render(<NoResult title={''} />);
        expect(container).toMatchSnapshot();
    });
});
