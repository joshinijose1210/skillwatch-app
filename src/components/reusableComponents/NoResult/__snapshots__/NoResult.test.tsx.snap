// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`KpiDescriptionEditor component should take snapshot 1`] = `
<div>
  <h5
    class="sc-1ntgl3l-0 kNdyZX"
    textalign="center"
    textcolor="inherit"
    textvariant="h5"
  />
</div>
`;
